app/views/admin/base.py

```diff
@@ -61,13 +62,34 @@ def user(self):
         def user(self, value):
             self._user = value
 
+    def _get_user_admin_permission(self):
+        user_repo_admin_object = admin_acls_model.AdminPermission.get(
```

Comment:
would .get() raise an exception if the object is not found? (instead of returning None) - would .filter(...).first() be more robust?

Reply:
`get()` is our function, and it calls `get_by_key_name` under the hood, which returns None if nothing's found:
https://cloud.google.com/appengine/docs/standard/python/datastore/modelclass#Model_get_by_key_name
This is a common case in tests (since typically only one or the other of the repo and global permissions will exist), so even if the `get()` implementation gets changed someday we'll catch it.